theme-switcher/switcher.js

```diff
@@ -0,0 +1,21 @@
+var themeList = document.getElementById('theme-list');
+
+function enableTheme(e) {
+  browser.management.setEnabled(e.target.value, true);
+  e.preventDefault();
+  window.close();
+}
+
+browser.management.getAll().then((themes) => {
+  for (let theme of themes) {
```

Comment:
This works, but... I thought it was weird to see `management.getAll()` only returning themes. It's incompatible with Chrome and is generally a surprising API.

Is it the intention with `getAll()` to never support other types, or is this just a first step? I saw [bug 1336908](https://bugzilla.mozilla.org/show_bug.cgi?id=1336908), but it didn't really elaborate on that.

I guess this is mostly a comment on the API rather than on this example, but maybe consider filtering the result:

    themes = themes.filter((e)=>{return e.type==="theme"})

That would make it more obvious and Chrome-compatible (maybe?) and would keep the code working in case we ever changed the behavior of getAll().

Or maybe it would be more confusing this way, since it would imply that we can return other types :).